ChangeLog.txt

```diff
@@ -46,7 +46,10 @@ MINOR:
 - Relatives of rulers can now join Root West Holy Order
 - Redface characters can now expel Root West Holy Order
 - Root West Holy Order now has some epic camels
-- Wahhabi Islam flavor expansion
+- Insular characters are now more likely to send missionaries to foreigners
```

Comment:
@loup99 

It's in the Religion section. I used an increase to "AI_ZEAL" to force them to send Missionaries more often, and added the Always Convert Flag.